### ISSUE TITLE
Calling prov.json before prov.capture

### ISSUE THREAD

**Issue by blernermhc** [ProvTools/provR#64]
If prov.json is called but prov.capture has not been run yet, there is a json string returned but it is not valid.  prov.json should probably return NULL in that case so that it is easy for the caller to check this case.

**Comment by MKLau** [comment]
Yeah, agreed. There should be a NULL message. Something like, "No provenance has been captured yet."

**Comment by blernermhc** [comment]
I would suggest returning NULL rather than a specific string value.  It is much easier to test for NULL.

**Comment by MKLau** [comment]
Good point, this shouldn't be human human readable anyway. I was thinking from the perspective of cleanR and not provR.
